guides/source/developers/customizations/customizing-permissions.html.md

```diff
@@ -0,0 +1,126 @@
+access control logic being scattered throughout the application source code.
+Instead, it enables one to define all user permissions in Ability classes.
+
+Solidus takes this concept to the next level, by allowing to specify
+different permission sets classes, and assign them to specific user roles.
+
+Let's see in details what these concepts represents and how to manage them.
+
+## Roles
+
+Solidus comes with two pre-configured roles: *admin* and *default*. User that
+`has_spree_role?(:admin)` has access to the admin panel and can manage all
+resources. User that `has_spree_role?(:default)` resembles a client or a website
+visitor, that can view certain resources, manage their shopping carts, etc.
+Each user can have multiple roles and admin users can change the roles of other
+users using the Admin panel, under the Users section.
+
+## Permission sets
+
+Solidus comes with a list of ready permission sets, that you can find into
+[core/lib/spree/permission_sets][permissions-sets], and a `Spree.config.roles`
+preference, that you can use to change or extend default permission sets.
+
+By default, roles and permission sets are configured with the following
+[code][roles-configuration]:
+
+```ruby
+# core/lib/spree/app_configuration.rb
+
+# ...
+
+Spree::RoleConfiguration.new.tap do |roles|
+  roles.assign_permissions :default, ['Spree::PermissionSets::DefaultCustomer']
+  roles.assign_permissions :admin, ['Spree::PermissionSets::SuperUser']
+end
+
+# ...
+```
+
+Which maps a list of permission sets to each role that we can use:
+
+- Users with role `default` will take
+  [`Spree::PermissionSets::DefaultCustomer`][default-customer-permissions] permissions
+- Users with role `admn` will take
+  [`Spree::PermissionSets::SuperUser`][admin-permissions] permissions
+
+
+[permissions-sets]: https://github.com/solidusio/solidus/tree/master/core/lib/spree/permission_sets
+[roles-configuration]: https://github.com/solidusio/solidus/blob/3e6de0ce0c190fd7415d46557da5786c4dda13dd/core/lib/spree/app_configuration.rb#L445-L450
+[default-customer-permissions]: https://github.com/solidusio/solidus/blob/master/core/lib/spree/permission_sets/default_customer.rb
+[admin-permissions]: https://github.com/solidusio/solidus/blob/master/core/lib/spree/permission_sets/super_user.rb
+
+
+## Manage Roles
+
+If we want to add a new role with its own set of permissions to our store
+the first thing to do is creating a new `Spree::Role`, which can be done
+with one of the following options:
+
+- Manually add a row in the `spree_roles` table by executing
+`Spree::Role.create(name: 'role_name')` in the Rails console;
+- In one of the configuration files (`config/intializers/spree.rb`,
+  `config/application.rb`, `db/migrations`, `db/seeds`) add a line
+  `Spree::Role.find_or_create_by(name: 'role_name')` for each role you wish
+  to create.
+
+Now that the new role has been created you can simply assign a new list of
+permission set to it, in the Solidus initializer:
```

Comment:
Since we are talking about a list, this should be plural.
`permission set` -> `permission sets`